index.html

```diff
@@ -136,22 +140,24 @@
         // prepend http: to prevent invalid schemes like file:// or uri://
         const getImageUrl = (person) => {
             return `http:${person.headshot.url}`;
+            //
```

Comment:
cache headshot images so they don't have to load each time the visibleArray changes